src/java/org/apache/cassandra/db/marshal/VectorType.java

```diff
@@ -352,10 +352,10 @@ private void check(List<?> values)
         }
     }
 
-    private static  <V> void checkConsumedFully(V buffer, ValueAccessor<V> accessor, int offset)
+    private <V> void checkConsumedFully(V buffer, ValueAccessor<V> accessor, int offset)
     {
         if (!accessor.isEmptyFromOffset(buffer, offset))
-            throw new MarshalException("Unexpected extraneous bytes after vector value");
+            throw new MarshalException("Unexpected extraneous bytes after " + asCQL3Type() + " value");
```

Comment:
Can we give more details about the  left size from offset using : `default int sizeFromOffset(V value, int offset)
    {
        return size(value) - offset;
    }`

Reply:
I'm not sure I see a lot of value in printing the specific size of the remaining bytes, and we don't do it in similar methods, but I have added it nevertheless.